l10n_co_res_partner/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright (C) Dominic Krimmer (Plastinorte S.A.S).
+# Author        Dominic Krimmer, dominic.krimmer@gmail.com
+# Co-Authors    Hector Ivan Valencia Muñoz
```

Comment:
Add the shor liocence format, and put your names in the Contributors File, more visibility and less python lines (even if comments it is cleanest to read).

when you make a blame with the commit history it is possible to know who write what, then and it is so much reliable.